plugins/python/plang/Environment.cpp

```diff
@@ -66,14 +66,37 @@ EnvironmentPtr Environment::get()
     return &g_environment;
 }
 
+
+#if (PY_VERSION_HEX >= 0x03000000)
+int wake_up_numpy()
+#else
+void wake_up_numpy()
+#endif
+{
+    import_array();
+#if (PY_VERSION_HEX >= 0x03000000)
+    return 0;
+#else
+#endif
+}
+
```

Comment:
I'm not sure what you mean? We still need to have import_array happen somewhere in the plugin, right? The problem is Python 3's import_numpy is a macro that just returns crap, and our clang settings error out in that situation. I put it in its own function in case we ever need to do something else.